master/networking/openstack/semantics.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Detailed semantics
+description: Describes connectivity that {{site.prodname}} provides, how and why this connectivity is different from traditional Neutron API semantics.
```

Comment:
I didn't think this worked

Reply:
Last I checked with Neil a few months ago, he said he believed the OpenStack content was accurate and up to date. I took this description directly from the doc. If it is not, correct maybe we should open separate PR to review the doc?

Reply:
Fixed